articles/vmware/vmw-how-create-firewall-rules.md

```diff
@@ -59,6 +59,9 @@ To create a firewall rule:
 
 6. For the **New Rule**, specify a **Name**.
 
+> [!NOTE]
+> The Firewall rules above are just an example we do not advise you use Any, Any, Any, Accept.
```

Comment:
firewall does not need to have an uppercase F here.
Also, I think the language should be a bit stronger for this statement, something like:
The firewall rules above provide a simple example; you should not use a firewall rule with Any, Any, Any, Accept as this allows all traffic through.
Or:
The firewall rule above provide a simple example; we strongly advise that you do not use Any, Any, Any, Accept as this allows all traffic through.

Reply:
Updated with second suggestion